src/swnr.pcss

```diff
@@ -28,7 +28,8 @@ a:hover {
   @apply text-shadow-gray-600;
 }
 div.editor .editor-content{
-  min-height: 200px;
+  min-height: 100px;
+  resize: vertical;
```

Comment:
Don't update or remove data unrelated to PR

Reply:
Done as part of #37.